extensions/objects/templates/real-editor.component.ts

```diff
@@ -35,8 +37,33 @@ export class RealEditorComponent implements OnInit {
     type: 'float'
   };
 
+  onChange: (_: unknown) => void = () => {};
```

Comment:
Should there be a TODO for removing this `unknown`?

Reply:
Done!